README.md

```diff
@@ -66,18 +66,83 @@ If you need to set a default configuration
 $ spack install celeritas +cuda cuda_arch=80
 ```
 
-
-Then see the "Downstream usage as a library" section of the [installation
-documentation][install] for how to use Celeritas in your application or framework.
-
 [spack-start]: https://spack.readthedocs.io/en/latest/getting_started.html
 [install]: https://celeritas-project.github.io/celeritas/user/usage/installation.html
 
+# Integrating into a Geant4 app
+
+In the simplest case, integration requires a small change to your project's
+CMake file:
+```diff
++find_package(Celeritas 0.6 REQUIRED)
+ find_package(Geant4 REQUIRED)
+@@ -36,3 +37,4 @@ else()
+   add_executable(trackingmanager-offload trackingmanager-offload.cc)
+-  target_link_libraries(trackingmanager-offload
++  celeritas_target_link_libraries(trackingmanager-offload
++    Celeritas::accel
+     ${Geant4_LIBRARIES}
+```
+
+a few includes:
+```diff
+--- example/accel/trackingmanager-offload.cc
++++ example/accel/trackingmanager-offload.cc
+@@ -36,2 +36,10 @@
+
++// Celeritas
++#include <accel/AlongStepFactory.hh>
++#include <accel/SetupOptions.hh>
++#include <accel/TrackingManagerConstructor.hh>
++#include <accel/TrackingManagerIntegration.hh>
++
++using TMI = celeritas::TrackingManagerIntegration;
++
+ namespace
+```
+
+minor additions to the user actions:
```

Comment:
Consider extending the description with a very small explanation of 'why' those changes are needed and/or what they do.  In particular so the user can decide where inside their `BeginOfRunAction` the code belongs (beginning, end or ... middle) or even if they need to create one if they don't have any.

Reply:
I don't know if the readme needs to handle that level of complexity. Is referring to the docs enough?

Reply:
Well the part where they need to decide whether to put the Celeritas code in beginning, middle or end of their own customization .... even it it is to say it doesn't matter :)